### PR TITLE
[ID] Add "Storyboarding" and "Storyboarder"

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
Add new Indonesian translation of Storyboarding and Storyboarder wiki articles.

### CODE REVIEW

File: wiki/Storyboarding/id.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Storyboarding
+
+<!-- TODO: Needs to fit in with storyboard scripting somehow -->
+
+<!-- Two important links are wiki/Beatmap_Editor/Design (visual) and wiki/Storyboard_Scripting (manual coding done by KJ) -->
+
+## Membuat sebuah storyboard
+
+[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (shortcut keyboard F2) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap sebagai kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard layar lebar, Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```
Comment (wowcake) [inline wiki/Storyboarding/id.md:13]:
```suggestion
[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (shortcut keyboard F2) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap sebagai kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```

**Review by wowcake — CHANGES_REQUESTED** [review]
looks good to me honestly

File: wiki/Storyboarding/id.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Storyboarding
+
+<!-- TODO: Needs to fit in with storyboard scripting somehow -->
+
+<!-- Two important links are wiki/Beatmap_Editor/Design (visual) and wiki/Storyboard_Scripting (manual coding done by KJ) -->
+
+## Membuat sebuah storyboard
+
+[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (shortcut keyboard F2) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap sebagai kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```
Comment (mousesalt) [inline wiki/Storyboarding/id.md:13]:
```suggestion
[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (dapat diakses dengan shortcut F2 pada keyboard) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```

File: wiki/Storyboarding/id.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Para pengguna yang berpengalaman dalam pembuatan storyboard sering menggunakan backend code yang dibuat oleh SBE, berjudul [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS) sebagai gantinya, yang memberikan mereka kontrol lebih banyak. Lihat artikel tersebut untuk detil lebih lanjut.
```
Comment (mousesalt) [inline wiki/Storyboarding/id.md:15]:
```suggestion
Pengguna yang ahli dalam pembuatan storyboard sering kali menggunakan backend code yang dibuat oleh SBE, berjudul [Storyboard Scripting](/wiki/Storyboard_Scripting) (SBS), dikarenakan kode ini menawarkan mereka kontrol lebih banyak. Lihat artikel tersebut untuk detil lebih lanjut.
```

Comment (mousesalt) [reply wiki/Storyboarding/id.md:15]: `Para` already say as many similar to `pengguna`, would be confused if there double negative

File: wiki/Storyboarding/Storyboarder/id.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - SBer
+---
+
+# Storyboarder
+
+**Storyboarder** adalah istilah yang digunakan untuk mendeskripsikan pengguna yang membuat storyboard untuk [beatmap](/wiki/Beatmap) milik sendiri atau orang lain. Storyboarder menggunakan editor beatmap pada bagian Desain, aplikasi pihak ketiga, atau [storyboard scripting](/wiki/Storyboard_Scripting) untuk membuat [storyboards](/wiki/Storyboards).
```
Comment (mousesalt) [inline wiki/Storyboarding/Storyboarder/id.md:9]:
```suggestion
**Storyboarder** adalah istilah yang digunakan untuk mendeskripsikan pengguna yang membuat storyboard untuk [beatmap](/wiki/Beatmap) milik sendiri atau milik orang lain. Untuk membuat [storyboard](/wiki/Storyboards), storyboarder menggunakan editor beatmap pada bagian Desain, aplikasi pihak ketiga, atau [storyboard scripting](/wiki/Storyboard_Scripting).
```

Comment (mousesalt) [reply wiki/Storyboarding/Storyboarder/id.md:9]: pasif sentence more real to read, which addressing the act as the first

File: wiki/Storyboarding/id.md (hunk 2 above)
Comment (ZeroPyrozen) [reply wiki/Storyboarding/id.md]: Damn I double-type that `sebagai` hahaha

**Review by mousesalt — APPROVED** [review]
:+1: looks fine now

**Review by TicClick — APPROVED** [review]

**Pull request merged**